### PR TITLE
Added documentation for reading proeprties goal

### PR DESCRIPTION
**Pull request by mipo256**
Documentation #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,153 @@
...
+   ```
+   <files>
+       <file>/path/to/common/common.properties</file>
+       <file>/path/to/very/common/another.properties</file>
+       <file>/path/to/not/that/common/some.properties</file>
+   </files>
+   ```
+   These are concrete properties files that you would want to load properties from. **The properties will be loaded from each of this files
+   in the order they declared**. That means, properties from file `some.properties` in case of conflict will override properties from file
+   `another.properties`. So the order in which you specify files here matters. <br/>
```
Comment (serezakorotaev) [inline README.md:75]: Maybe **matters** ?)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,153 @@
...
+1. Load properties into maven build from external sources - goal `read-project-properties`.
+2. Write the properties used in maven build to files - goals `write-project-properties` and `write-active-profile-properties`.
+
+Let's explore each of this goals one by one.
+
+# 2. Goal `read-project-properties`
+
+<h3>2.1 Common usage</h3>
+
+As I said, this goal is used to load properties from different sources into Maven build. The simple usage will look like this:
```
Comment (serezakorotaev) [inline README.md]: I think if you use "we" in all text, then here also "we" instead "i"

File: README.md [hunk 3]
```diff
@@ -1,4 +1,153 @@
...
+   as we have for `files`. I.e. if you specified multiple URLs, then properties files would be loaded from each of these URLs, but the latter 
+   URLs properties can override the properties from the former URLs.
+3. Files by pattern
+   ```
+   <includes>
+       <include>/home/user/props/*.properties</include>
+       <include>/etc/service/configs/**/*.properties</include>
+   </includes> 
+   ```
+   Here, we specified, that any file, that has `.properties`  extension and is located at `` is a properties file that should be loaded.
```
Comment (serezakorotaev) [inline README.md]: Perhaps here symbol `` is odd?

**Review by serezakorotaev — COMMENTED** [review]
good job with documentation, Mikhail! But 3 small issue i wrote here)

File: README.md (hunk 2 above)
Comment (mipo256) [reply README.md]: Agree

File: README.md (hunk 3 above)
Comment (mipo256) [reply README.md]: Not, it is not is not odd, it's just my mistake, thanks for noting

File: README.md (hunk 1 above)
Comment (mipo256) [reply README.md:75]: I do not understand, what do you mean?)

Comment (serezakorotaev) [reply README.md:75]: Bold for users)

**Review by serezakorotaev — APPROVED** [review]
All is okay)

**Pull request merged**